backend/pick-git/src/main/java/com/woowacourse/pickgit/post/domain/PostRepository.java

```diff
@@ -1,10 +1,20 @@
 package com.woowacourse.pickgit.post.domain;
 
 import com.woowacourse.pickgit.user.domain.User;
+import java.util.List;
 import java.util.Optional;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface PostRepository extends JpaRepository<Post, Long> {
 
     Optional<Post> findByUser(User user);
+
+    @Query("select p from Post p left join fetch p.user order by p.id")
```

Comment:
```suggestion
    @Query("select p from Post p left join fetch p.user order by p.id desc")
```

내림차순으로 조회해야 최신순으로 불러올 수 있을 거 같아요!

Reply:
아 근데 비로그인 상태에서 아직 어떤 기준으로 조회할지 안 정했으니까 지금은 이대로 가도 괜찮겠네요 !!

Reply:
이 부분은 홈 피드(전체 게시물 조회)에 관련한 쿼리인 거죠?

Reply:
이 메소드를 홈 피드 + 프로필 피드 모두 사용합니다!